libs/sdk-ui-charts/src/highcharts/chartTypes/_chartOptions/chartOptionsBuilder.ts

```diff
@@ -324,6 +331,7 @@ export function getChartOptions(
     dataView: IDataView,
     chartConfig: IChartConfig,
     drillableItems: IHeaderPredicate[],
+    emptyHeaderName: string,
```

Comment:
should not be named as emptyElement/Value? It represents single element, not whole attributeHeader, in not it?

Reply:
In fact it is a header title related to the given value so it is a header. I changed the naming slightly and updated the TSDocs.